packages/ckeditor5-table/src/tablecolumnresize/tablecolumnresizeediting.js

```diff
@@ -581,12 +598,18 @@ export default class TableColumnResizeEditing extends Plugin {
 				}
 			} else {
 				// In read-only mode revert all changes in the editing view. The model is not touched so it does not need to be restored.
+				// This case can occur if the read-only mode kicks in during the resizing process.
 				editingView.change( writer => {
 					if ( isColumnWidthsAttributeChanged ) {
-						const columnWidths = columnWidthsAttributeOld.split( ',' );
+						if ( columnWidthsAttributeOld ) {
```

Comment:
Let's add a short comment what is it needed for.

Reply:
I added a comment - this `if` is necessary to avoid running the `split()` method if there is no previous value (so it would result in an error). I see that I didn't add a test for this case here, but in a CC fixing PR - https://github.com/ckeditor/ckeditor5/pull/12049/files#diff-d08d4a33389c9c57bdda1abc834750454c8f1d7f135bee1dd9fa4ff5a13eb883R888-R907. I'd rather not copy this one test to avoid issues during branch merging - do you think we can do it like that?

Reply:
Ok, #12049 was just merged, so we have this test here already :see\_no\_evil: